sources/parsers/auto/DecimalParser.cpp

```diff
@@ -1209,5 +1209,6 @@ namespace PARSER {
 void PARSER::DecimalParser::error(const DecimalParser::location_type& l,
                                   const std::string& m)
 {
+    (void)m;
```

Comment:
Remove param name instead of void casting.

Reply:
It seems that the parser generator needs this syntax.